server/routes/mention.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const MentionController = require("../Controllers/Mention.Controller");
+
+router.get("/", MentionController.getAllMentions);
```

Comment:
We can actually move the MentionController code into the routes here, sorry for the confusion.